src/components/player.vue

```diff
@@ -480,6 +480,7 @@ export default {
 		play () {
 			this.playingComputed = true
 			this.started = true
+			this.$emit('update:played', this.playing, this.$el);
```

Comment:
i dont think the element is needed as a closure can made the callback of the emit. Will make a PR correcting this.